ui/src/app/lib/atlasmap-data-mapper/services/mapping-management.service.ts

```diff
@@ -549,7 +556,7 @@ export class MappingManagementService {
   }
 
   private hasMultipleMappings(mapping: MappingModel): boolean {
-    return (mapping.sourceFields.length > 2 || mapping.targetFields.length > 2);
+    return (mapping.sourceFields.length > 1 || mapping.targetFields.length > 1);
```

Comment:
Checking for multiple mappings needs to account for the None field which occupies sourceFields[0] or targetFields[0] if in one-to-manmy or many-to-one.  I other words I think you were okay at '2'

Reply:
The problem is when there are arrays or collections involved, the none field isn't always present, so I have to account for that.

Reply:
Need to test the case where a user was in a multiple-mode and then removed their multiplicity to re-establish a onetoone mapping (ctrl-select on one of the previously establishes mappings).  This would trigger at 2 in the case

Reply:
I also think this is unrelated to #905, please remove this change.